cmd/sd/internal/help/format.go

```
@@ -0,0 +1,103 @@
+	}
+	return 80
+}
+
+// SetupMarkdownHelp configures a cobra command to render its Long description
+// with beautiful markdown formatting in the terminal.
+func SetupMarkdownHelp(cmd *cobra.Command) {
+	cmd.SetHelpFunc(func(c *cobra.Command, args []string) {
+		// Render the Long description with glamour
+		if c.Long != "" {
+			formatted := FormatMarkdown(c.Long, c.OutOrStdout())
+			fmt.Fprint(c.OutOrStdout(), formatted)
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)